lib/lunchbot_web/live/select_menu_live.ex

```diff
@@ -6,18 +6,20 @@ defmodule LunchbotWeb.Admin.SelectMenuLive do
   alias Lunchbot.LunchbotData.OfficeLunchOrder
 
   def mount(_params, _session, socket) do
+    olos = LunchbotData.list_most_recent_office_lunch_orders()
     changeset = LunchbotData.change_office_lunch_order(%OfficeLunchOrder{})
     offices = LunchbotData.list_office_name_id_tuples()
     menus = LunchbotData.list_menu_name_id_tuples()
 
     socket =
       assign(socket,
+        olos: olos,
         changeset: changeset,
         offices: offices,
         menus: menus
       )
 
-    {:ok, socket}
+    {:ok, socket, temporary_assigns: [olos: []]}
```

Comment:
Very nice.  Thanks for the explanation:  Using the phx-update event with a value of "prepend" in the heex template allows new olos to be prepended to the dom, so that the list of olos in memory from the initial render may be reset to an empty list (transitory), there-by freeing up memory.